ReactViewResources/Loader/Loader.ts

```diff
@@ -17,6 +17,7 @@ const reactLib: string = "React";
 const reactDOMLib: string = "ReactDOM";
 const viewsBundleName: string = "Views";
 const pluginsBundleName: string = "Plugins";
+const defaultStyleSheetId = "default-stylesheet";
```

Comment:
store the dom element instead of having an id that could cause clash with user dom